Route/RoutePageGenerator.php

```diff
@@ -148,11 +160,18 @@ public function update(SiteInterface $site, OutputInterface $output = null)
 
             $page->setSlug($route->getPath());
             $page->setUrl($route->getPath());
-            $page->setRequestMethod(isset($requirements['_method']) ? $requirements['_method'] : 'GET|POST|HEAD|DELETE|PUT');
+            $page->setRequestMethod(isset($requirements['_method']) ?
+                $requirements['_method'] :
+                'GET|POST|HEAD|DELETE|PUT');
```

Comment:
No it shouldn't : the opening parenthesis isn't. The only argument to this call is multiline. Multiline arguments != multiline function call.